prisma/seed.ts

```diff
@@ -0,0 +1,95 @@
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+import { PrismaClient, Prisma } from '@prisma/client';
+import * as faker from 'faker';
+
+const prisma = new PrismaClient();
+
+const main = async () => {
```

Comment:
This is much closer to what I meant, but I still think the actual hardcoded data should not be in this file, it should still be an external `.json`, is that not possible?

Like it should be:
```ts
PUZZLE_TYPES.forEach(puzzleType => {
  const puzzleType = await prisma.puzzleType.create({
    data: {
      name: puzzleType.name
    }
  });
};
```

Where `PUZZLE_TYPES` is an array of objects defined in a separate file and imported here

Reply:
Yep yep!